pom.xml

```diff
@@ -68,6 +65,16 @@
 	</dependencyManagement>
 
 	<dependencies>
+		<dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter</artifactId>
+			<version>5.4.2</version>
```

Comment:
Looks like you removed the version for this `<junit-jupiter.version>5.4.2</junit-jupiter.version>`

Reply:
thanks, resolved